Server/src/services/dataExtraction/FileHandlerFactory.ts

```diff
@@ -12,7 +12,7 @@ export abstract class AbstractFileExtractor {
     constructor(filePath: string) {
         this.filePath = filePath
     }
-    protected abstract parseFile()
+    abstract parseFile()
```

Comment:
Why was protected dropped here?

Reply:
Imo, this is abstract and protecting abstract functions limit us in using the function in other places, like Unit Tests for example.

Reply:
Ah yeah if you're testing the function directly then that makes sense.